features/foutafhandeling.feature

```diff
@@ -0,0 +1,135 @@
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        | detail | Bad request.                                  |
+        En bevat de response de volgende invalidParams
+        | name | code           | reason                                             |
+        | bbox | surfaceMaximum | Waarde is hoger dan maximum oppervlakte 250000 m2. |
+
+Rule: min waarde mag niet groter zijn dan max waarde
+
+    Abstract Scenario: opgegeven minimale waarde is groter dan opgegeven maximale waarde
+        Als '<path><query string>' wordt aangeroepen
+        Dan bevat de response de volgende velden
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        | detail | Bad request.                                  |
+        En bevat de response de volgende invalidParams
+        | name             | code   | reason                |
+        | <parameter>[min] | <code> | <reason omschrijving> |
+
+        Voorbeelden:
+        | path                   | query string                               | parameter   | code  | reason omschrijving              |
+        | /adresseerbareobjecten | ?oppervlakte[min]=200&oppervlakte[max]=100 | oppervlakte | range | min mag niet hoger zijn dan max. |
+        | /panden                | ?bouwjaar[min]=2000&boujaar[max]=1000      | bouwjaar    | range | min mag niet hoger zijn dan max. |
+
+Rule: waarde van parameter in request body is valide
+
+    Abstract Scenario: waarde van een parameter in de request body is niet correct
+        Als '<path><request body>' wordt aangeroepen
+        Dan bevat de response de volgende velden
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        | detail | <detail>                                      |
+        En bevat de response de volgende invalidParams
+        | name        | code   | reason                |
+        | <parameter> | <code> | <reason omschrijving> |
+
+        Voorbeelden:
+        | path      | request body                                                                                                                             | parameter            | code             | reason omschrijving                      |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((a b,c d))                                                                                       | geometrie.intersects | geometryFormat   | Waarde is niet conform {format} formaat. |
```

Comment:
In de reason omschrijving moet je hier de template invullen. Dus niet "Waarde is niet conform {format} formaat.", maar "Waarde is niet conform Well Known Text (WKT) formaat.". Anders verwacht ik ook echt letterlijk die tekst met {format} inclusief de accolades.

Reply:
Aangepast